Added remaining vendor inventories

Peptide90 opened the pull request:
Made remaining vendors usable.
nutri and robotics have some job specific items missing that aren't in yet (nutri is two tools, robotics is surgical related).

medical will need updated when medical is in.

Theater I put some costumes in that we already have in. Nowhere near as exhaustive as /TG/ but some of the other items are already covered in the shoe and hat vendor for example.

Untouched are Ptech (Cart) for PDA cartridges, sales as this is generic and doesn't need inventory? Vendomat and Vox.
I also noticed TG had wardrobes with all job specific clothing items in so could be worth considering as well as a games vendor.


For the most part will roundout and close #436 minus the exceptions above which are as and when.

Zumorica commented on Resources/Prototypes/Catalog/VendingMachines/nutri.yml:
```diff
@@ -3,3 +3,14 @@
+    PlantBGoneSpray: 20
+    WeedSpray: 20
+    PestSpray: 20
+    #syringe
```
We already have syringes, the prototype ID is just `Syringe`

Peptide90 replied:
Added thanks, wasn't sure if they were in and working.

Zumorica commented on Resources/Prototypes/Catalog/VendingMachines/robotics.yml:
```diff
@@ -3,3 +3,20 @@
+    #signaller
+    #health anaylzer
+    Scalpel: 2
+    #circular saw
```
The prototype ID for this one is `BoneSaw`, pretty sure

Peptide90 replied:
I'll add that in, wasn't sure if bonesaw was a hand saw rather than the circular saw. Thanks

mirrorcult commented on Resources/Prototypes/Catalog/VendingMachines/nutri.yml:
```diff
@@ -3,3 +3,14 @@
+    WeedSpray: 20
+    PestSpray: 20
+    #syringe
+    #cultivator
```
should add like "TODO: add missing prototypes" to the files that have stuff commented out so it's easier to find them

SweptWasTaken commented:
Address reviews and this can be merged.

These will be populated over time as we get more content, so don't feel bad about commenting stuff in.

Peptide90 commented:
> Address reviews and this can be merged.
> 
> These will be populated over time as we get more content, so don't feel bad about commenting stuff in.

Added in the missing stuff from comments and added a To Do comment where there was more than one thing missing. Thanks

PaulRitter commented on Resources/Prototypes/Catalog/VendingMachines/robotics.yml:
```diff
@@ -3,3 +3,21 @@
+    #signaller
+    #health anaylzer
+    Scalpel: 2
+    BoneSaw
```
```suggestion
    BoneSaw: 1
```

PaulRitter reviewed (changes requested):
fix it & get merged
